internal/agent/list.go

```diff
@@ -58,13 +58,12 @@ func NewAgentList(loader func(int) tea.Cmd, page, perpage int) AgentListModel {
 func (m *AgentListModel) appendAgents() tea.Cmd {
 	// Set agentsLoading
 	m.agentsLoading = true
-	// Set a status message and start the agentList's spinner
-	startSpiner := m.agentList.StartSpinner()
+	// Set a status message
 	statusMessage := fmt.Sprintf("Loading more agents: page %d of %d", m.agentCurrentPage, m.agentLastPage)
 	setStatus := m.agentList.NewStatusMessage(statusMessage)
```

Comment:
Looks like `quiet` is not propagated here (and probably other places aside from `SpinWhile`. Might be a good idea to add a flag to the `api` layer to suppress the output.